packages/tools/src/utilities/contours/updateContourPolyline.ts

```diff
@@ -14,6 +14,12 @@ import {
  * @param viewport - Viewport
  * @param polylineData - Polyline data (points, winding direction and closed)
  * @param transforms - Methods to convert points to/from canvas and world spaces
+ * @param options - Options
```

Comment:
I don't see any tool using this option. I expected the simplify object being part of the tools configuration?

Reply:
Updated spline, freehand and livewire configs